### DISCUSSION TITLE
Vertipaq Analyzer - Read Statistics from Data

### DISCUSSION THREAD

**Discussion by SharmaAntriksh** [DaxStudio/DaxStudio#713] [Q&A] [answered]
I just wanted to know what is the actual use of the _**Read statistics from data option**_ in the Advanced tab under File > Options. I know there is a proper definition available under it which is self sufficient in understanding it, but I am not able to see any visual changes in the View Metrics option, I might be missing something.

![image](https://user-images.githubusercontent.com/73054674/133423918-d9670c53-20c1-4278-9fa6-294b73874dd0.png)

**Comment by marcosqlbi** [comment, marked as answer]
There are cases where the numbers returned by the DMV are not correct.
Just as an examples (not covering all the possible cases): the number of unique values reported by the DMVs include items no longer used in a column, whereas the number returned by "read statistics from data" is more accurate and ignores the number of items in the dictionary that are no longer used in the columns. This condition can only happen after incremental and/or partition refresh, after a full refresh of a table the two number should always match.
